service/debugger/debugger.go

```diff
@@ -798,6 +867,7 @@ func (d *Debugger) FindBreakpoint(id int) *api.Breakpoint {
 	d.targetMutex.Lock()
 	defer d.targetMutex.Unlock()
 	bps := api.ConvertBreakpoints(d.findBreakpoint(id))
+	bps = append(bps, d.findDisabledBreakpoint(id))
```

Comment:
Should only do the append if the result of `findDisabledBreakpoints` is not nil.